lib/gui-utils-oled/src/GUIUtils.cpp

```diff
@@ -69,6 +69,9 @@ void GUIUtils::showWelcome() {
     u8g2.sendBuffer();
 }
 
+void GUIUtils::PowerSave() {
+    u8g2.setPowerSave(1);
```

Comment:
maybe is possible using a internal constant on the u8g2 library that resolve this `1` value? only for better understanding of the code

Reply:
1 = deep sleep enable
0 = Display On, deep sleep disable
I  don't know how else to do it, sorry.